@shared/api/types.ts

```diff
@@ -43,6 +43,7 @@ export interface Response {
 export interface ExternalRequest {
   transactionXdr: string;
   network: string;
+  accountToSign: string;
```

Comment:
`accountToSign` is what I'm calling the public key that the application is specifically looking for